ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -77,6 +99,20 @@ tests:
   skip_if_only_changed: (^(docs|examples|enhancements|contrib)/)|(^[A-Z]+\.md$)|((^|/)OWNERS$)
   steps:
     workflow: hypershift-aws-e2e-nested
+- as: e2e-aws-4-12
+  optional: true
```

Comment:
They are optional to avoid overload the CI Root cluster? or they will be optional always and execute them in periodics?

Reply:
right, this is to avoid overloading our CI as this will introduce 2 new pipelines which is roughly 20 more e2e clusters. We don't want to run that for each PR.